src/xml_document.cc

```diff
@@ -244,17 +246,25 @@ NAN_METHOD(XmlDocument::FromHtml)
     xmlResetLastError();
     xmlSetStructuredErrorFunc(reinterpret_cast<void*>(&errors), XmlSyntaxError::PushToArray);
 
+
+    static const int DEFAULT_PARSING_OPTS = 0;
```

Comment:
Seems to me like they're unlikely to be reused, but I think they'd fit better as static members of the class. If they are moved, header will need to include "libxml/HTMLparser.h"